easybuild/easyconfigs/m/MEGAHIT/MEGAHIT-1.1.2-foss-2018a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'MEGAHIT'
+version = '1.1.2'
+
+homepage = 'https://github.com/voutcn/megahit'
+description = """An ultra-fast single-node solution for large and complex
+ metagenomics assembly via succinct de Bruijn graph"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/voutcn/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d0d3965dd49c6fdaea958ef66146cb6b30b7d51acbbfe94194c437f15a424cb5']
+
+# This version is CPU-only.  It's also possible to compile this with
+# CUDA, but that isn't working yet (and requires libcuda.so!).
+
+dependencies = [
+    ('Python', '2.7.14'),
+]
+
+# for GPU, 'use_gpu=1 abi=1' at least, plus CUDA dep above
+buildopts = 'version=v%(version)s verbose=1 use_gpu=0'
```

Comment:
This looks a bit strange, what would happen if you don't include the `version=v%(version)s` part?
Maybe it's worth clarifying that with a comment above this line?

Reply:
In the `Makefile`, `version` will be set to `git describe --tag`.  I'm not sure that even works for release tarballs.  Even if it does, though, it potentially fails badly if the commit that has a release tag also has other tags.  Since we *know* the version (since we're using it to pull the same tag from github), it seems safest to specify it explicitly.  Will add a comment.